plutus-contract/src/Plutus/Contract/Test/ContractModel/MissingLovelace.hs

```diff
@@ -0,0 +1,30 @@
+module Plutus.Contract.Test.ContractModel.MissingLovelace
```

Comment:
Why put it as a submodule of `ContractModel`?

Reply:
I decided to move this function to its own module because `ContractModel.Internal` is already almost 2k lines. Also because I wanted to write some long comment to it and a separate module might emphasize that this function is special in some sense. But I don't mind can move it to `Internal`.

Reply:
Sorry, I was unclear. What I meant was: Why as a submodule of `ContractModel` specifically? `calculateDelta` can be used outside of ContractModel, no? Maybe put it `Plutus.Contract.Test.MissingLovlace`? I could be wrong...